### DISCUSSION TITLE
Different fonts used when changing List name

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#14120] [Mobile] [answered]
If I go to change a list name and I start typing without moving around, new text will use a different font. If I move around, it will take over the first font.

![image](https://user-images.githubusercontent.com/63184600/161402460-231239b7-9aab-4b3e-87fa-88b6c68f5415.jpeg)

Don't know if this also happens in other places.

iOS 15.4.1, GitHub app 1.58.0

**Comment by iskae** [comment, marked as answer]
Hi @amadejpapez, thanks so much for the report! We are able to reproduce the issue 👍  A ticket has been created and we will be fixing it in our upcoming release.
